lib/package/sqs/compileIamRoleToSqs.js

```diff
@@ -1,20 +1,22 @@
 'use strict'
 const _ = require('lodash')
 
+const SERVICE_NAME = 'sqs'
+
 module.exports = {
   compileIamRoleToSqs() {
+    if (!this.shouldCreateDefaultRole(SERVICE_NAME)) {
+      return
+    }
+
     const sqsQueueNames = this.getAllServiceProxies()
-      .filter((serviceProxy) => this.getServiceName(serviceProxy) === 'sqs')
+      .filter((serviceProxy) => this.getServiceName(serviceProxy) === SERVICE_NAME)
       .map((serviceProxy) => {
         const serviceName = this.getServiceName(serviceProxy)
         const { queueName } = serviceProxy[serviceName]
         return queueName
       })
 
-    if (sqsQueueNames.length <= 0) {
```

Comment:
why was this removed?

Reply:
See comment here https://github.com/horike37/serverless-apigateway-service-proxy/pull/42#discussion_r327597590